jenkins/trigger-build/jenkins_trigger_builds.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+# This script triggers the YaST job builds for the master branch at 
+# the publis Jenkins.
+#
+# Pass the Jenkins credentials via "JENKINS_USER" and "JENKINS_PASSWORD"
+# environment variables.
+
+if !ENV["JENKINS_USER"] || !ENV["JENKINS_PASSWORD"]
+  $stderr.puts "Error: The jenkins credentials are not set."
+  $stderr.puts "Pass them via the 'JENKINS_USER' and 'JENKINS_PASSWORD' " \
+    "environment variables."
+  exit 1
+end
+
+# install missing gems
+if !File.exist?("./.vendor")
+  puts "Installing needed Rubygems to ./.vendor/bundle ..."
+  system("bundle install --path .vendor/bundle")
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require "jenkins_api_client"
+require "logger"
+
+JENKINS_URL = "https://ci.opensuse.org"
+
+puts "Reading Jenkins jobs from #{JENKINS_URL}..."
+jenkins = JenkinsApi::Client.new(server_url: JENKINS_URL, log_location: "jenkins.log",
+  username: ENV["JENKINS_USER"], password: ENV["JENKINS_PASSWORD"])
+
+# get only the master branch YaST jobs
+jenkins_jobs = jenkins.job.list_all.select { |j| j.match(/^yast|^libyui/) && j.end_with?("-master")}
```

Comment:
No, the skelcd is [yast-skelcd-control-openSUSE-master](https://ci.opensuse.org/view/Yast/job/yast-skelcd-control-openSUSE-master/), the reason was to avoid running some docker or maintenance Jobs.